system/Autoloader/Autoloader.php

```diff
@@ -71,10 +71,19 @@ class Autoloader
     /**
      * Stores files as a list.
      *
-     * @var array<int, string>
+     * @var string[]
+     * @phpstan-var list<string>
      */
     protected $files = [];
 
+    /**
+     * Stores helper list.
+     *
+     * @var string[]
+     * @phpstan-var list<string>
+     */
+    protected $helpers = [];
```

Comment:
Let's consume our own feature:
```suggestion
    protected $helpers = ['url'];
```
Then use `array_merge()` with the Config and drop the helper call from bootstrap.php

Reply:
Done.